uber/models.py

```diff
@@ -648,6 +648,9 @@ def change_badge(self, attendee, badge_type, badge_num=None):
         def valid_attendees(self):
             return self.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS)
 
+        def all_attendees(self):
+            return self.staffers(only_staffing=False)
+
```

Comment:
Ahh, I see. I was reading the docstring for `staffers`, which says:
>However, on-site we relax that restriction, so we'll get attendees with shifts who are not actually marked as staffing.

That strongly indicates that it pulls people with shifts, but then as you have proven, it totally doesn't. :\ Annoying.

If you could include a docstring that explains that we're using that particular method only because it sideloads the other data, or possibly rename it to `all_attendees_jobs_groups` or something, that'd be ideal, but I no longer feel super strongly about it one way or the other.